understanding/20/error-prevention-all.html

```diff
@@ -8,6 +8,14 @@
 <body>
    <h1>Understanding Error Prevention (All)</h1>
    
+		<section id="brief">
+			<h2>In brief</h2>
+			<dl>
+				<dt>Objective</dt><dd>Users can avoid submitting incorrect information</dd>
+				<dt>Author task</dt><dd>Provide ways for users to confirm, correct, or reverse all submissions</dd>
```

Comment:
I was trying to differentiate the similar AA from the AAA requirement, but I agree "all" is cumbersome. How about
```suggestion
				<dt>Author task</dt><dd>Provide ways for users to confirm, correct, or reverse any submissions</dd>
```